x-pack/solutions/chat/plugins/workchat-app/public/application/components/chat/conversations_panel/agent_block.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { css } from '@emotion/css';
+import {
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  useEuiTheme,
+  EuiHealth,
+  EuiAvatar,
+  EuiSkeletonText,
+} from '@elastic/eui';
+import { useAgent } from '../../../hooks/use_agent';
+
+interface AgentBlockProps {
+  agentId: string;
+}
+
+export const AgentBlock: React.FC<AgentBlockProps> = ({ agentId }) => {
```

Comment:
nit: We are diverging with naming: assistant<-> agent? If we refer to agents as assistant in UI, should the new code also reference assistant instead agent?

Reply:
Very fair point, will rename